src/raw/mod.rs

```diff
@@ -75,16 +75,17 @@ impl<K, V> Table<K, V> {
     pub(crate) fn get_moved<'g>(
         &'g self,
         for_table: Shared<'g, Table<K, V>>,
-        guard: &'g Guard,
+        guard: &'g Guard<'_>,
     ) -> Shared<'g, BinEntry<K, V>> {
         match self.next_table(guard) {
             t if t.is_null() => {
                 // if a no next table is yet associated with this table,
                 // create one and store it in `self.next_table`
-                match self.next_table.compare_and_set(
+                match self.next_table.compare_exchange(
                     Shared::null(),
                     for_table,
                     Ordering::SeqCst,
+                    Ordering::SeqCst,
```

Comment:
I _think_ the fallback here can be `Relaxed`

Reply:
Fixed in 011a9df178983d95a9f6d43cfc404576904107be.